fund_store/config/fund_loader_config/FAB/gbrf_r1.py

```diff
@@ -0,0 +1,87 @@
+LOADER_CONFIG = {
+    "sections_config": [
+        {
+            "section_name": {"en": "1. Local authority details", "cy": ""},
+            "tree_path": "1050.1.1",
+            "requires_feedback": None,
+        },
+        {
+            "section_name": {"en": "1.1 Local authority details", "cy": ""},
+            "tree_path": "1050.1.1.1",
+            "form_name_json": {"en": "local-plans-local-authority-details-signed-off", "cy": ""},
+        },
+        {"section_name": {"en": "2. Spending proposals", "cy": ""}, "tree_path": "1050.1.2", "requires_feedback": None},
+        {
+            "section_name": {"en": "2.1 Spending proposals", "cy": ""},
+            "tree_path": "1050.1.2.1",
+            "form_name_json": {"en": "green-belt-spending-proposals", "cy": ""},
+        },
+        {"section_name": {"en": "3. Declarations", "cy": ""}, "tree_path": "1050.1.3", "requires_feedback": None},
+        {
+            "section_name": {"en": "3.1 Declarations", "cy": ""},
+            "tree_path": "1050.1.3.1",
+            "form_name_json": {"en": "green-belt-declarations", "cy": ""},
+        },
+    ],
+    "fund_config": {
+        "id": "f97e3930-ab32-4353-84a6-3053d05382ae",
+        "short_name": "GBRF",
+        "welsh_available": False,
+        "owner_organisation_name": "None",
+        "owner_organisation_shortname": "None",
+        "owner_organisation_logo_uri": "None",
+        "ggis_scheme_reference_number": "G2-SCH-2023-03-12555",
+        "funding_type": "COMPETITIVE",
+        "name_json": {"en": "Green Belt Review Funding", "cy": ""},
+        "title_json": {"en": "Green Belt Funding", "cy": ""},
```

Comment:
for DPIF we have used `funding to begin your digital planning improvement journey` so dont we need similar title or are we good with this ?

Reply:
So.. initially it was all lowercase. But we display the `title_json` in a few places and I thought all lowercase looked ugly.